docs/snippets/vue/list-story-reuse-data.js.mdx

```diff
@@ -0,0 +1,20 @@
+```js
+// List.stories.js
+
+import List from './ListComponent.vue';
+import ListItem from './ListItem.vue';
+
+import { Selected, Unselected } from './ListItem.stories';
+
+export const ManyItems = (args, { argTypes }) => ({
+  components: { List, ListItem },
+  props: Object.keys(argTypes),
+  template: `
+    <list v-bind="$props">
+      <list-item :itemProperty="Selected"/>
+      <list-item :itemProperty="Unselected"/>
+      <list-item :itemProperty="Unselected"/>
```

Comment:
How does this work? How does the string turn into the variable here? Am I missing something obvious?

Reply:
It's injected from the `args` which was included with the latest commit.